Automatic Validation ended with:
> 2025-02-05T15:54:30.0642731Z ##[error] Manifest Error: The multi file manifest has inconsistent field values. [ManifestVersion] Value: 1.6.0 File: Python.Python.3.12.locale.zh-CN.yaml
 2025-02-05T15:54:30.0967729Z ##[error] Manifest Error: The multi file manifest has inconsistent field values. [ManifestVersion] Value: 1.6.0 File: Python.Python.3.12.locale.zh-CN.yaml

(Automated response - build 976.)

--- a/manifests/p/Python/Python/3/12/3.12.9/Python.Python.3.12.locale.zh-CN.yaml
+++ b/manifests/p/Python/Python/3/12/3.12.9/Python.Python.3.12.locale.zh-CN.yaml
@@ -1,5 +1,5 @@
 # Created with YamlCreate.ps1 Dumplings Mod
-# yaml-language-server: $schema=https://aka.ms/winget-manifest.locale.1.6.0.schema.json
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.locale.1.9.0.schema.json
 
 PackageIdentifier: Python.Python.3.12
 PackageVersion: 3.12.9
@@ -32,4 +32,4 @@ Documentations:
 - DocumentLabel: 文档
   DocumentUrl: https://www.python.org/doc/
 ManifestType: locale
-ManifestVersion: 1.6.0
+ManifestVersion: 1.9.0

